pkg/object/unstructured_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	cmdtesting "k8s.io/kubectl/pkg/cmd/testing"
-	. "sigs.k8s.io/cli-utils/pkg/object"
+	. "sigs.k8s.io/cli-utils/pkg/object" //nolint:revive
```

Comment:
Are we using a lot of package vars?
Probably better to call them all with the package scope than just disable the linter. 

Adding nolint to ginkgo and gomega is fine, those are canonically dot imports. But we should avoid it most other places unless there's a great reason.

Reply:
Ok, I'll remove the dot import for this one (in all files).

Reply:
Done.